lib/elixir_analyzer/exercise_test.ex

```diff
@@ -76,8 +76,17 @@ defmodule ElixirAnalyzer.ExerciseTest do
 
     {ast, block_params} =
       case ast do
-        {:__block__, _, params} -> {params, length(params)}
-        _ -> {ast, false}
+        {:__block__, _, params} ->
+          case length(params) do
+            1 ->
+              {hd(params), false}
+
+            len ->
+              {params, len}
+          end
+
+        _ ->
+          {ast, false}
```

Comment:
It makes my tests work and doesn't break anything else as far as I can tell. I think we will have to write analyzers for a few more exercises to really test it.

Style-wise, I would propose to remove the nested case and do pattern-matching instead:
```suggestion
        {:__block__, _, [param]} ->
          {param, false}

        {:__block__, _, [_ | _] = params}  ->
          {params, length(params)}

        _ ->
          {ast, false}
```

Reply:
ya, I just wanted to make sure that it actually worked before worrying too much about aesthetic

Reply:
I like the suggestion tho